bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckRun.java

```diff
@@ -1370,11 +1370,12 @@ private void checkStatus() {
     }
 
     private boolean isFileUpdated(String filename, Hash hash) throws IOException {
-        return localRepo.commits(hash.hex(), 1).stream()
-                .anyMatch(commit -> commit.parentDiffs().stream()
-                        .anyMatch(diff -> diff.patches().stream()
-                                .anyMatch(patch -> (patch.source().path().isPresent() && patch.source().path().get().toString().equals(filename))
-                                        || ((patch.target().path().isPresent() && patch.target().path().get().toString().equals(filename))))));
+        return localRepo.show(Path.of(filename), hash).isPresent() &&
```

Comment:
We can make this check a bit cheaper by calling `!localRepo.files(hash, Path.of(filename)).isEmpty()` instead.

Reply:
Yes, it will be cheaper.  Thanks!